src/messageforwarder.py

```diff
@@ -0,0 +1,241 @@
+import discord
```

Comment:
Very minor: I'm not crazy about the file name

Reply:
The age old problem of naming things :P

Any suggestions? I'm fine with whatever.

Perhaps `dm.py`? Also if you'd want to change the class name too then, lmk.

Reply:
I'm fine with the class name as is, perhaps just `forwarder.py`? I think it would be fine with just a single word.